### PR TITLE
Remove Stripe Proxy

### PR DESCRIPTION
**Pull request by farski**
This also removes the LegacySupport stack since Stripe Proxy was the only thing in there, and templates without resources can't be deployed.

Closes https://github.com/PRX/internal/issues/588

### CODE REVIEW

File: stacks/container-apps/sphinx-nlb.yml [hunk 1]
```diff
@@ -474,7 +474,7 @@ Resources:
           TargetGroupArn: !Ref NLBSphinxTargetGroup
       # Role: !Ref ECSServiceIAMRole
       NetworkConfiguration:
-        AwsvpcConfiguration:
+        AwsVpcConfiguration:
```
Comment (farski) [inline stacks/container-apps/sphinx-nlb.yml:477]: This either changed, or cfn-lint was allowing both and now isn't.

**Review by cavis — APPROVED** [review]
👍

File: stacks/container-apps/sphinx-nlb.yml (hunk 1 above)
Comment (kookster) [reply stacks/container-apps/sphinx-nlb.yml:477]: That seems to be right now though
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-networkconfiguration.html

**Review by kookster — APPROVED** [review]
lgtm

**Pull request merged**